applications/KZero.md

```diff
@@ -136,6 +136,50 @@ In short, the Kzero process can be broken down into the following steps
 
 
 
+### Recovery Mechanisms
+
+There are currently two potential, optional recovery mechanisms available:
+
+1. **Built-in recovery mechanism in zklogin-pallet**: Each zklogin address can designate a recoverer. Under specific conditions, such as when a zklogin address has been inactive for N hours, the recoverer can send transactions and manage assets on behalf of the zklogin address.
+2. **zklogin + proxy**: Through the proxy pallet, a zklogin address can set up 'delegate' or 'delayed-delegate' accounts. At any time when assets in the zklogin address face potential threats, users can immediately handle their assets securely through the proxy module.
```

Comment:
Sounds good, thanks for adding.
Could you clarify whether this is within the scope of this grant?

Reply:
@takahser Given that these two mechanisms are not in conflict, we prioritize implementing the built-in recovery mechanism in this grant, while planning to develop the zkLogin + proxy mechanism at the application level in the future.

Reply:
@dejavukong sounds good to me. Could you add this to the deliverables (or in case it's already added, point out which deliverable this would equate to)?

After that I'm happy to approve it.

Reply:
@takahser included in Milestone 2